domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SampleClaim.java

```diff
@@ -32,7 +32,7 @@
     public static final String DEFENDANT_ID = "4";
     public static final Long CLAIM_ID = 3L;
     public static final String REFERENCE_NUMBER = "000CM001";
-    public static final String EXTERNAL_ID = UUID.randomUUID().toString();
+    public static final UUID EXTERNAL_ID = UUID.randomUUID();
```

Comment:
This change requires changing the tests in core module of claim-store. Failing test compilation due to this